pkg/virt-handler/migration-target.go

```diff
@@ -493,11 +513,6 @@ func (c *MigrationTargetController) execute(key string) error {
 		return nil
 	}
 
-	if !c.vmiExpectations.SatisfiedExpectations(key) {
-		log.Log.V(4).Object(vmi).Info("waiting for expectations to be satisfied")
-		return nil
-	}
-
```

Comment:
Was that for debugging purposes or do we actually want to get rid of expectations in this controller?